bin/appmetrics-cli.js

```diff
@@ -49,3 +49,7 @@ if (process.env.NODE_PATH) {
 }
 
 var p = child_process.fork(launcher_filename, args, {"execArgv": node_args})
+
+p.on('close', (code) => {
```

Comment:
This will work if the child process exits, but if it terminates with signal, the cli will exit with `0`.

```
on('close', (code, signal) => {
  if(signal) {
    process.kill(process.pid, signal);
  } else {
    process.exit(code);
  }
});
```
might work better